pages/_app.tsx

```diff
@@ -46,18 +47,32 @@ if (appConfig.useAxe === "true" && !isServerRendered()) {
 
 function MyApp<MyAppProps>({ Component, pageProps, userLocation, query }) {
   useRouterScroll({ top: 640 });
+  const formInitialStateCopy = { ...formInitialState };
   const formMethods = useForm<FormInputData>({ mode: "onBlur" });
   // TODO: Work on CSRF token auth.
   const csrfToken = "";
   const { favIconPath, appTitle } = appConfig;
+
+  // TODO: Display the errors for individual pages.
+  let errors;
+  if (!isEmpty(query.errors)) {
+    errors = JSON.parse(query.errors);
```

Comment:
For the next PR. We'll just display the errors at the top of each page when js is not available.